invoke/program.py

```diff
@@ -86,6 +87,12 @@ def core_args(self) -> List["Argument"]:
                 default=False,
                 help="Echo executed commands before running.",
             ),
+            Argument(
+                names=("suggestions", "s"),
+                kind=bool,
+                default=True,
```

Comment:
Is it possible to default here to True?

If someone wanted to disable new feature, it is possible with --no-suggestions I guess.